src/KeyVault/Microsoft.Azure.KeyVault.WebKey/JsonWebKey.cs

```diff
@@ -0,0 +1,637 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+
+namespace Microsoft.Azure.KeyVault.WebKey
+{
+    using System;
+    using System.Linq;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using System.Runtime.Serialization;
+    using System.Security.Cryptography;
+
+    /// <summary>
+    /// As of http://tools.ietf.org/html/draft-ietf-jose-json-web-key-18
+    /// </summary>
+    public class JsonWebKey
```

Comment:
This looks like a generated model.  If it is, it should go in the Generated folder